andela/titan-dev-ah-frontend
# 163518638 Configure TravisCI

#### What does this PR do?
This PR sets up the TravisCI and simple test
#### Description of Task to be completed?
* We created the`.travis.yml` file that contains every configuration required.
#### How should this be manually tested
* Visit travis-ci.org 
* Click the button to sign in with Github. 
* Give Travis CI access to your account. 
* You can specify exactly which repositories Travis CI should be given access to.
* From there it will be either `Red Color` to indicate The failure or The `Green color` to indicate the success of integration between the project and TravisCI

#### Any background context you want to provide?
None  
#### What are the relevant pivotal tracker stories?
#163518638 
#### Questions:
None
